### PR TITLE
Adds defines for species and bodytypes

### PR DESCRIPTION
**Pull request by Cyantime**
Replaces all bare strings for species names and bodytypes with defines.

This should contain zero actual changes, with the exception of the `/datum/ghostspawner/human/kataphract/klax` spawner, which was checking the wrong value for whitelists, and the silver material now makes silver golems instead of bronze ones.

### CODE REVIEW

File: code/__defines/species.dm [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#define SPECIES_HUMAN				"Human"
```
Comment (NonQueueingMatt) [inline code/__defines/species.dm:1]: Very slight nitpick that doesn't have to be addressed, but evening out the spacing here would make my eyes feel nice.

Comment (Cyantime) [reply code/__defines/species.dm:1]: The spacing is consistent when viewed in dreammaker.

File: code/game/machinery/suit_cycler.dm [hunk 2]
```diff
@@ -113,39 +113,39 @@
...
+	species = list(SPECIES_HUMAN, BODYTYPE_TAJARA, BODYTYPE_SKRELL, BODYTYPE_UNATHI, BODYTYPE_IPC)
 	can_repair = TRUE
 
 /obj/machinery/suit_cycler/freelancer
 	name = "freelancers suit cycler"
 	model_text = "Freelancers"
 	req_access = list(access_distress)
 	departments = list("Freelancers")
-	species = list("Human", "Tajara", "Skrell", "Unathi", "Machine")
+	species = list("Human", BODYTYPE_TAJARA, BODYTYPE_SKRELL, BODYTYPE_UNATHI, BODYTYPE_IPC)
```
[the remark this answers is not in the document]
Comment (Cyantime) [reply code/game/machinery/suit_cycler.dm:148]: Whoops, guess I missed one

**Comment by Cyantime** [comment]
Current travis failure is unrelated to changes, some sort of seed generation error.

**Pull request merged**